web/src/test/java/org/springframework/security/web/authentication/AbstractAuthenticationTargetUrlRequestHandlerTests.java

```diff
@@ -67,27 +67,37 @@ void returnDefaultTargetUrlIfUseDefaultTargetUrlTrue() {
 
```

Comment:
Hi @jzheaux , although it somewhat fit for a parameterised test, I think separate test methods make more sense here, what do you think ?